rc/.eslintrc

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["standard", "standard-react"]
+  "extends": ["standard", "standard-react"],
+  "rules": {
+    "standard/object-curly-even-spacing": [2, "either"],
```

Comment:
We could put these in the `eslint-config-standard` but then other would have to `require('eslint-plugin-standard')` to use it without it blowing up. Might be nice to get feedback from consumers of `eslint-config-standard` such as @Flet